apps/src/p5lab/AnimationTab/PiskelEditor.jsx

```diff
@@ -239,7 +249,7 @@ class PiskelEditor extends React.Component {
 }
 export default connect(
   state => ({
-    selectedAnimation: state.animationTab.selectedAnimation,
+    currentAnimations: state.animationTab.currentAnimations,
```

Comment:
right now, `currentAnimations` comes from redux (here) and `currentAnimationType` is always a prop (not in redux). however, if both pieces of data were in redux, we could simplify the logic in this component because the `connect` function could calculate the animation on its own

```javascript
currentAnimation: state.animationTab.currentAnimations[state.animationTab.currentAnimationType]
```

would this simplify logic elsewhere, too? if it would, this might be a valuable refactor. if not, we could add a helper function or something like that to still simplify without adding state to redux

Reply:
Done! 
bb3febf47bb4fd54f0b6fc061519f2fa2b6a7268
bb3febf47bb4fd54f0b6fc061519f2fa2b6a7268

This was a fun refactor. TIL a potential advantage of redux state. I've often had to pass a React property down through many components even though only a couple actually use the value. With Redux, I can use `connect` to just bring the data I care about directly into the components that need it. It keeps the other components feeling cleaner too!